src/components/Books/Books.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import NewBook from '../NewBook/NewBook';
+import {
+  BookWrapper, BookDivLeft, BookDivRight,
+  BookDivCenter, ButtonsContainer, Gauage, PercentageSpan, StatusSpan,
+  Category, Title, Author, BooksButton, CurrentChapter,
+  Chapter, UpdateButton,
+} from './BooksElements';
+
+const Books = () => (
+  <>
+    <BookWrapper>
+      <BookDivLeft>
+        <Category>Action</Category>
+        <Title>The Hunger Games</Title>
+        <Author>Suzanne Collins</Author>
```

Comment:
- [ ] Let's create a new component called `BookList` that renders a list of books using the 'Books' component.
- [ ] Let's pass the book data as a prop to the `Books` component and render the title and the author

Reply:
I totally agree with this :) Tbh, I planned to do that in the next step but doing it at the beginning looks like a best practice. Thank you :)